tests/nominal/test_OneHotEncodingTransformer.py

```diff
@@ -264,3 +265,76 @@ def test_unseen_categories_encoded_as_all_zeroes(self, df_test, expected):
             actual=df_transformed,
             msg="unseen category rows not encoded as 0s",
         )
+
+
+class TestAdditionalOneHotEncodingTransformer:
+    """Additional tests for OneHotEncodingTransformer."""
+
+    @pytest.fixture
+    def sample_data(self):
+        return pd.DataFrame(
+            {
+                "color": ["red", "blue", "green", "blue", "red"],
+                "size": ["S", "M", "L", "S", "M"],
+            },
+        )
+
+    def test_not_fitted_error_transform(self, sample_data):
```

Comment:
duplicate of GenericNominalTransformTests.test_not_fitted_error_raised

Reply:
removed